README.md

```diff
@@ -40,3 +40,21 @@ Contributions are very welcome, as are feature requests and suggestions. Please
 [codecov-url]: https://codecov.io/gh/JuliaData/CSV.jl
 
 [issues-url]: https://github.com/JuliaData/CSV.jl/issues
+
+## Alternatives
+
+There are several other packages for reading CSV files in Julia, which may suit your needs better:
+
+* The standard library contains [DelimitedFiles.jl](https://docs.julialang.org/en/v1/stdlib/DelimitedFiles/).
+  This returns a `Matrix` rather than a [Tables.jl](https://github.com/JuliaData/Tables.jl)-style container, thus works best for files of homogenous element type. 
+  On large files, CSV.jl will be much faster.
+
+* [CSVFiles.jl](https://github.com/queryverse/CSVFiles.jl) uses the [FileIO.jl](https://github.com/JuliaIO/FileIO.jl) API 
+  into any [IterableTables.jl](https://github.com/queryverse/IterableTables.jl) sink.
+  The package uses [TextParse.jl](https://github.com/queryverse/TextParse.jl) for parsing.
+
+* [DLMReader.jl](https://github.com/sl-solution/DLMReader.jl) also aims to be fast for large files,
+  closely associated with [InMemoryDatasets.jl](https://github.com/sl-solution/InMemoryDatasets.jl).
```

Comment:
> DLMReader.jl is a great reference because it supports some more exotic parsing configurations if you need to get really custom.

Should this link say something like "exotic custom parsing"?